apps/microsoft-teams/frontend/src/hooks/useGetTeamsChannels.ts

```diff
@@ -38,7 +39,7 @@ const useGetTeamsChannels = () => {
     getAllChannels();
   }, [getAllChannels]);
 
-  return channels;
+  return import.meta.env.DEV ? mockChannels : channels;
```

Comment:
this is intentional at the moment to be able to work in development with the hosted app action call.. If we don't like this I can definitely remove it. What are your toughts @jsdalton, @lilbitner, @rusticpenguin ?

Reply:
yeah I think this work - maybe worth adding a note above to adjust eventually?